gril/ril_constants.h

```diff
@@ -427,4 +465,37 @@
 #define RIL_REG_STATE_EMERGENCY_DENIED 13
 #define RIL_REG_STATE_EMERGENCY_UNKNOWN 14
 
+/* Radio capability phases */
+#define RIL_RC_PHASE 0
+#define RIL_RC_PHASE_CONFIGURED 1
```

Comment:
These enums seem to be off by 1 compared the values in AOSP v6 version of ril.h ( version 11 ).